src/main/scala/io/keen/client/scala/Client.scala

```diff
@@ -361,6 +366,23 @@ trait Writer extends AccessLevel {
    */
   val writeKey: String = settings.writeKey.getOrElse(throw MissingCredential("Write key required for Writer"))
 
+  // constants
+  val MinSendInterval: Integer = 60
+  val MaxSendInterval: Integer = 3600
+
+  // initialize and configure our local event store queue
+  val batchSize: Integer = settings.batchSize.getOrElse(500)
+  val batchTimeout: Integer = settings.batchTimeout.getOrElse(5)
+  val eventStore: RamEventStore = new RamEventStore
+  eventStore.maxEventsPerCollection = settings.maxEventsPerCollection.getOrElse(10000)
```

Comment:
Seems like the type of `eventStore` should be the more general `EventStore` or similar, specialized to `RamEventStore` as a default. `maxEventsPerCollection` would need to be defined on the trait…